lightkurve/tests/test_correctors.py

```diff
@@ -150,3 +151,19 @@ def test_to_corrector():
     tpf = KeplerTargetPixelFile(TABBY_TPF)
     lc = tpf.to_corrector("pld").correct()
     assert len(lc.flux) == len(tpf.time)
+
+
+@pytest.mark.remote_data
+@pytest.mark.skipif(bad_optional_imports, reason="PLD requires celerite and fbpca")
+def test_pld_aperture_mask():
+    """Regression test for #523: does PLDCorrector.correct() accept separate
```

Comment:
Note: most people wouldn't call this a regression test because there hasn't been a regression i.e. bug here, this is simply a new feature.